governance/api.go

```diff
@@ -167,10 +167,8 @@ type VoteList struct {
 func (api *PublicGovernanceAPI) MyVotes() []*VoteList {
 
 	ret := []*VoteList{}
```

Comment:
Isn't it better to set the size of the slice? 
Like `ret := make([]*VoteList{}, api.governance.voteMap.Size())`

Reply:
@ehnuje It is interesting. `ret := []*VoteList{}` and `ret := make([]*VoteList{}, api.governance.voteMap.Size())` shows different results when it is printed in console.

When your suggestion applied, the result is like below. It adds additional null in slot 0.
```shell
> governance.myVotes
[null, {
    BlockNum: 236,
    Casted: true,
    Key: "reward.ratio",
    Value: "100/0/0"
}]
```

Reply:
How about `ret := make([]*VoteList{}, 0, api.governance.voteMap.Size())`

Reply:
https://play.golang.org

Reply:
This is because the size and the capacity have the same value. As we use append here, 0 should be given for the size.

Reply:
@ehnuje Done. Thanks!